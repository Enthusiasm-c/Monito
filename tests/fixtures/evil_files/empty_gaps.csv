Название,,Категория,Unnamed_4,Описание
,,,,
iPhone 14,99999,Смартфоны,Apple,Новый iPhone
,,,,
Samsung S23,89999,,Samsung,Galaxy S23
MacBook Pro,200000,Ноутбуки,,
,,,,
