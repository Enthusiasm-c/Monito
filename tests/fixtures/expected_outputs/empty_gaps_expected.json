{
  "filename": "empty_gaps.csv",
  "expected_rows_in": 3,
  "expected_rows_out": 3,
  "challenges": [
    "Empty rows",
    "Empty column headers",
    "Missing data",
    "Unnamed columns"
  ]
}